tests/integrated/test-multigrid_laplace_sheared-grid/data/BOUT.inp

```diff
@@ -0,0 +1,115 @@
+grid = "grids/grid_MAST_SOL_jyis4.nc"
+mz = 128
+dump_format = "nc"
+
+[mesh]
+myg = 0
+#nx = 132
```

Comment:
Do the commented out lines have a particular significance? If so is it worth adding a comment to note this, if not is it worth removing the commented lines?